### PR TITLE
fix: rename libqt5-charts key to libqt5-charts-dev to match the other Qt5 -dev packages

### PR DESCRIPTION
**Pull request by esteve**
Please add the following dependency to the rosdep database.

## Package name:

libqt5charts5-dev

## Package Upstream Source:

See https://doc.qt.io/qt-5/qtcharts-index.html

## Purpose of using this:

It is useful when we develop rviz plugin with qt charts.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/bookworm/libqt5charts5-dev
- Ubuntu: https://packages.ubuntu.com/
  - https://packages.ubuntu.com/search?keywords=libqt5charts5-dev&searchon=names&suite=all&section=all
- Fedora: https://packages.fedoraproject.org/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL
- openSUSE: https://software.opensuse.org/package/
  - IF AVAILABLE
- rhel: https://rhel.pkgs.org/
  - IF AVAILABLE

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME

# The source is here:

http://sourcecode_url

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Review by audrow — COMMENTED** [review]
Seems reasonable, but this could have unintended consequences, like breaking CI for people that use it.

I'll make a note to discuss this with the ROS 2 team on Tuesday.

**Comment by satoshi-ota** [comment]
Hi @esteve Thanks but I've already added it in https://github.com/ros/rosdistro/pull/39705.

**Comment by esteve** [comment]
@satoshi-ota Thanks, but the point of this PR is to update the changes made in your PR so that `libqt5-charts-dev` follows the same naming convention as the other development packages. Please check my comment in your previous PR https://github.com/ros/rosdistro/pull/39705#issuecomment-1926588319 for context.

File: rosdep/base.yaml [hunk 1]
```diff
@@ -5294,7 +5294,7 @@ libqt4-sql-psql:
   macports: [qt4-mac]
   nixos: [qt4]
   ubuntu: [libqt4-sql-psql]
-libqt5-charts:
+libqt5-charts-dev:
```
Comment (clalancette) [inline rosdep/base.yaml:5297]: I think it is a good idea to add in the `-dev` suffix; that way in the future we could add in a runtime library.

@satoshi-ota Given that you originally added this key in https://github.com/ros/rosdistro/pull/39705, are you OK if we rename it here?

Comment (satoshi-ota) [reply rosdep/base.yaml:5297]: @clalancette No problem. Thank you :+1:

**Review by clalancette — APPROVED** [review]

**Review by sloretz — APPROVED** [review]

**Pull request merged**